src/routes/transactions/updateTransaction.ts

```diff
@@ -75,10 +76,12 @@ router.patch(
         `Transaction "${transactionId}" could not be found.`
       );
     }
-    const transactionProperty = transaction.property as PropertyDocument;
+    let transactionProperty = transaction.property as PropertyDocument;
```

Comment:
This change from `const` to `let` probably has no effect.

Reply:
Thank you Jimmy!